File: BackEnd/Models/Classes/BonusQuestion.cs
using BackEnd.Models.Enums;

namespace BackEnd.Models.Classes
{
    public class BonusQuestion
    {
        public int Id { get; set; }
        public string? Question { get; set; }
        public int PredictionGameId { get; set; }
        public BonusQuestionType QuestionType { get; set; }
        public string? CorrectAnswer { get; set; }
        public List<string>? Options { get; set; }
    }
}

File: Frontend/types/bonusQuestion.ts
export type BonusQuestionType = 'StringQuestion' | 'NumberQuestion' | 'MultipleChoiceQuestion';

export interface BonusQuestion{
    id: number,
    predictionGameId: number,
    question: string,
    questionType: BonusQuestionType,
    correctAnswer: string,
    options: string[],
}